src/DataObject/Attribute/Parameter/Query/ClassNameParameter.php

```diff
@@ -37,7 +37,7 @@ public function __construct()
             schema: new Schema(
                 type: 'string',
                 enum:  array_map(static fn (ClassDefinition $def) => $def->getName(), $classDefinitions->load()),
-                example: null
+                example: 'string'
```

Comment:
```suggestion
                example: '--'
```
I guess if the example is not in the enum, it will resort to null right?
Maybe we should use the '--' here since that how it is shown in the docs
![image](https://github.com/user-attachments/assets/9d706ba1-df49-48fc-85c5-41c5296215f9)

Maybe removing the example all together does work?